specs/middleware-injector.specs.ts

```diff
@@ -12,20 +12,27 @@ describe("middleware-injector", () => {
 
   const entriesInfo = {
     background: {name: "bgChunkName", path: "./path/to/bg-script.js"},
-    contentScript: {name: "contentChunkName", path: "./path/to/content-script.js"}
+    contentScript: {name: "contentChunkName", path: "./path/to/content-script.js"},
+    contentScript2: {name: "contentChunkName2", path: "./path/to/content-script2.js"}
   };
 
   const options: EntriesOption = {
     background: entriesInfo.background.name,
     contentScript: entriesInfo.contentScript.name
   };
 
+  const optionsMultipleCS: EntriesOption = {
+    background: entriesInfo.background.name,
+    contentScript: [entriesInfo.contentScript.name, entriesInfo.contentScript2.name]
+  };
+
   const fakeCssPath = "./path/to/some.css";
   const fakeImgPath = "./path/to/a/random-image.png";
 
   const assets = {
     [entriesInfo.background.path]: {source: () => "const bg = true;"},
     [entriesInfo.contentScript.path]: {source: () => "const cs = true;"},
+    [entriesInfo.contentScript2.path]: {source: () => "const cs = true;"},
```

Comment:
Just put a small difference for the first content script. This avoid mistakes as we need compare both old and new versions of _contentScript_ and _contentScript2_ (and currently both are the same). Just do:
```
() => "const cs2 = true;"
```